skietbaan_fe/src/components/CreateComp.js

```diff
@@ -1,63 +1,101 @@
 import React, { Component } from 'react';
-import { Jumbotron, Form, Button } from 'react-bootstrap';
+import { Jumbotron, Form, Button, Row, Col, Container } from 'react-bootstrap';
 import propTypes from 'prop-types';
 import { connect } from 'react-redux';
 import '../scss/createcomp.css';
 import { createcomp } from '../actions/competition.action';
-//import axios from 'axios';
 class CreateComp extends Component {
-	constructor() {
-		super();
-		this.state = { Name: '', Status: true };
+	constructor(props) {
+		super(props);
+		this.state = { Name: '', BestScoresNumber: 0, Status: true, 
+						validNumScore: true, validCompName: true 
+					};
 		//binding the onChange method to this commponents
 		this.onChange = this.onChange.bind(this);
 		this.onSubmit = this.onSubmit.bind(this);
+		this.onClick = this.onClick.bind(this);
+	}
+	onClick() {
+		Window.location = '/create';
 	}
 	/** A method that detects the change in the change in thw textfield */
 	onChange(e) {
 		this.setState({ [e.target.name]: e.target.value });
 	}
+	onChangeInt(e) {
```

Comment:
Space between declaration

Reply:
Hi Fred, please clarify what you mean by a space between declaration

Reply:
spaces between functions 
```
function cookies(){}
// the space I was refering to 
function eat(){}
```